tools/ci_build/github/apple/build_ios_framework.py

```diff
@@ -40,52 +40,45 @@ def _parse_build_settings(args):
     return build_settings
 
 
-def _build_package(args):
-    build_settings = _parse_build_settings(args)
-    build_dir = os.path.abspath(args.build_dir)
-
-    # Temp dirs to hold building results
-    intermediates_dir = os.path.join(build_dir, 'intermediates')
-    build_config = args.config
-    base_build_command = [sys.executable, BUILD_PY, '--config=' + build_config] + build_settings['build_params']
-
+# Build fat framework for all archs of a single sysroot
+# For example, arm64 and x86_64 for iphonesimulator
+def _build_for_ios_sysroot(build_config, intermediates_dir, base_build_command, sysroot, archs):
     # paths of the onnxruntime libraries for different archs
     ort_libs = []
     info_plist_path = ''
 
     # Build binary for each arch, one by one
-    for osx_arch in build_settings['build_osx_archs']:
-        sysroot = osx_arch['sysroot']
-        current_arch = osx_arch['arch']
+    for current_arch in archs:
         build_dir_current_arch = os.path.join(intermediates_dir, sysroot + "_" + current_arch)
         build_command = base_build_command + [
             '--ios_sysroot=' + sysroot,
             '--osx_arch=' + current_arch,
             '--build_dir=' + build_dir_current_arch
         ]
 
-        if args.include_ops_by_config is not None:
-            build_command += ['--include_ops_by_config=' + str(args.include_ops_by_config.resolve())]
+        print(build_command)
 
         # the actual build process for current arch
-        subprocess.run(build_command, shell=False, check=True, cwd=REPO_DIR)
+        # subprocess.run(build_command, shell=False, check=True, cwd=REPO_DIR)
```

Comment:
>#  [](http://example.com/codeflow?start=8&length=2)

uncomment this?

Reply:
Uncommented, also removed some other print(...)